src/main/PDF/PDF.java

```diff
@@ -30,6 +30,9 @@ public class PDF {
   @BsonProperty(value = "pdfType")
   private PDFType pdfType;
 
+  @BsonProperty(value = "idCategory")
+  private String idCategory;
```

Comment:
can we also add this field to the file in the fileDao?

Reply:
Done